packages/terra-tabs/src/common-tabs/HiddenTab.module.scss

```diff
@@ -124,4 +124,27 @@
       width: 100%;
     }
   }
+
+  .tabs-remove-button {
+    background-color: #fff;
+    //background-image: inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#4d5153" d="M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z"></path></svg>');
```

Comment:
Why is this commented out?

Reply:
This CSS is for Closed button , the commented part is not required have removed in new commit. https://github.com/cerner/terra-framework/commit/f4bd3b60b2862061fc633330bda690d094cd0ea6